### PR TITLE
Invoice Ninja

### PR DESCRIPTION
**Pull request by VeselaHouba**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [X] I have tested the template using the method described in README.md thoroughly
- [X] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [X] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [X] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [X] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/invoiceninja.yml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            notExposeAsWebApp: true
+
+    $$cap_appname-server:
+        restart: always
+        depends_on:
+            - $$cap_appname-app
+        environment:
+            APP_CONTAINER: srv-captain--$$cap_appname-app:9000
+        volumes:
+            - $$cap_appname-app-public:/var/www/app/public:ro
```
Comment (githubsaturn) [inline public/v4/apps/invoiceninja.yml:127]: Remove `:ro`, it's not consumed by CapRover.

File: public/v4/apps/invoiceninja.yml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        caproverExtra:
+            # Workaround for hard-coded container name in nginx config
+            dockerfileLines:
+                - FROM nginx
```
Comment (githubsaturn) [inline public/v4/apps/invoiceninja.yml:131]: Add a version tag here. No need to export it in variables, but at least we should fix this version tag.

File: public/v4/apps/invoiceninja.yml [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                - ADD https://raw.githubusercontent.com/invoiceninja/dockerfiles/master/config/nginx/in-vhost.conf /etc/nginx/templates/in-vhost.conf.template
```
Comment (githubsaturn) [inline public/v4/apps/invoiceninja.yml:132]: This is always pointing to the latest `master` file. Instead, use a fixed commit for this link.

File: public/v4/apps/invoiceninja.yml (hunk 1 above)
Comment (githubsaturn) [reply public/v4/apps/invoiceninja.yml:127]: Same with `:rw` in other lines.

**Review by githubsaturn — COMMENTED** [review]
Looks good, I'm excited for this!

Just to confirm, this is tested and it's working?

File: public/v4/apps/invoiceninja.yml (hunk 3 above)
Comment (VeselaHouba) [reply public/v4/apps/invoiceninja.yml:132]: Makes sense. If they do some breaking changes in next release, the template will need to be reviewed anyway.

**Comment by VeselaHouba** [comment]
> 

Yes, it seems to be working. I have only tested to the extent of creating example company and checking that my changes are persistent.

File: public/v4/apps/invoiceninja.yml (hunk 1 above)
Comment (VeselaHouba) [reply public/v4/apps/invoiceninja.yml]: fixed

File: public/v4/apps/invoiceninja.yml (hunk 2 above)
Comment (VeselaHouba) [reply public/v4/apps/invoiceninja.yml]: fixed to 1.21

File: public/v4/apps/invoiceninja.yml (hunk 3 above)
Comment (VeselaHouba) [reply public/v4/apps/invoiceninja.yml]: fixed to latest tag 5.3.34

**Comment by VeselaHouba** [comment]
Rebased & squashed. @githubsaturn It's all yours now :)

**Comment by githubsaturn** [comment]
Thanks! Looks great!

**Pull request merged**